internal/pod_models.go

```diff
@@ -550,6 +550,9 @@ func configureSidecarContainer(container *corev1.Container, initMode bool, proce
 		if cluster.Spec.Routing.PodIPFamily != nil {
 			sidecarArgs = append(sidecarArgs, "--public-ip-family")
 			sidecarArgs = append(sidecarArgs, fmt.Sprint(*cluster.Spec.Routing.PodIPFamily))
+			if *cluster.Spec.Routing.PodIPFamily == 6 {
+				sidecarArgs = append(sidecarArgs, "--bind-address", "[::]:8080")
```

Comment:
Can we add a comment above this arg why we need it?

Reply:
done